src/rdata/rfc1035/txt.rs

```diff
@@ -756,7 +756,7 @@ impl<Builder: OctetsBuilder + AsRef<[u8]> + AsMut<[u8]>> TxtBuilder<Builder> {
             Symbol::from_chars(&mut chars).map_err(E::custom)?
         {
             if len == 255 {
-                return Err(E::custom(CharStrError));
+                return Err(E::custom(CharStrError::default()));
```

Comment:
Isn't calling default on a unit struct a Clippy violation?

Reply:
I don’t think so – but keep in mind that `CharStrError` isn’t a unit type any more. You can’t do `CharStrError(())` here either because the inner unit is private.

In any case, this code has disappeared with an earlier PR.